benches/hex.rs

```diff
@@ -0,0 +1,68 @@
+use criterion::{black_box, criterion_group, criterion_main, Criterion};
+use rustc_hex::{FromHex, ToHex};
+
+const DATA: &[u8] = include_bytes!("../src/lib.rs");
+
+fn bench_encode(c: &mut Criterion) {
+    c.bench_function("hex_encode", |b| b.iter(|| black_box(hex::encode(DATA))));
```

Comment:
Note that criterion puts everything that is returned by the closure into a `black_box` already: https://docs.rs/criterion/0.2.11/src/criterion/lib.rs.html#335-345

So many of these `black_box`es can be removed. Of course, one might still prefer to keep them for explicitness.

Reply:
> Note that criterion puts everything that is returned by the closure into a `black_box` already: https://docs.rs/criterion/0.2.11/src/criterion/lib.rs.html#335-345
> 
> So many of these `black_box`es can be removed. Of course, one might still prefer to keep them for explicitness.

Thanks for your reviewing :)
I will remove that.